java/maven.hints/src/org/netbeans/modules/maven/hints/pom/ParentVersionError.java

```diff
@@ -108,6 +109,16 @@ public List<ErrorDescription> getErrorsForDocument(POMModel model, Project prj)
                         NbMavenProject nbprj = parentPrj.getLookup().lookup(NbMavenProject.class);
                         if (nbprj != null) { //do we have some non-maven project maybe?
                             MavenProject mav = nbprj.getMavenProject();
+                            if(declaredVersion.startsWith("${") && declaredVersion.endsWith("}")){
```

Comment:
can you take a look at `PomModelUtils` `#isPropertyExpression`, `#getProperty` etc? Class is in the same package.

Reply:
Sure updated the code to incorporate these methods from PomModelUtils.